modules/ServerAPI/src/server/routes/ExperimentServiceRoutes.coffee

```diff
@@ -490,11 +490,15 @@ exports.genericExperimentSearch = (req, res) ->
 			username = req.query.username
 		else
 			username = "none"
-		baseurl = config.all.client.service.persistence.fullpath+"experiments/search?q="+req.params.searchTerm+"&userName="+username
-		console.log "baseurl"
-		console.log baseurl
-		serverUtilityFunctions = require './ServerUtilityFunctions.js'
-		serverUtilityFunctions.getFromACASServer(baseurl, res)
+		authorRoutes = require './AuthorRoutes.js'
+		authorRoutes.allowedProjectsInternal req.user, (statusCode, allowedUserProjects) ->
+			_ = require "underscore"
+			allowedProjectCodes = _.pluck(allowedUserProjects, "code")
+			baseurl = config.all.client.service.persistence.fullpath+"experiments/search?q="+req.params.searchTerm+"&projects=#{allowedProjectCodes.join(',')}"
```

Comment:
Does this same refactor need to be done in the ProjectBrowser module's search method? That is the other place where we limit results back based on accessible projects.

Reply:
Yes it should and this will be resolved with #555